Sources/X509/CryptographicMessageSyntax/CMSSignerInfo.swift

```diff
@@ -33,20 +33,25 @@ import SwiftASN1
 /// then the `version` MUST be 1.  If the `SignerIdentifier` is `subjectKeyIdentifier`,
 /// then the `version` MUST be 3.
 /// - Note: At the moment we neither support `signedAttrs` (`SignedAttributes`) nor `unsignedAttrs` (`UnsignedAttributes`)
+@usableFromInline
 struct CMSSignerInfo: DERImplicitlyTaggable, Hashable {
+    @usableFromInline
     enum Error: Swift.Error {
         case versionAndSignerIdentifierMismatch(String)
     }
+
+    @inlinable
     static var defaultIdentifier: ASN1Identifier {
         .sequence
     }
     
-    var version: CMSVersion
-    var signerIdentifier: CMSSignerIdentifier
-    var digestAlgorithm: AlgorithmIdentifier
-    var signatureAlgorithm: AlgorithmIdentifier
-    var signature: ASN1OctetString
-    
+    @usableFromInline var version: CMSVersion
+    @usableFromInline var signerIdentifier: CMSSignerIdentifier
+    @usableFromInline var digestAlgorithm: AlgorithmIdentifier
+    @usableFromInline var signatureAlgorithm: AlgorithmIdentifier
+    @usableFromInline var signature: ASN1OctetString
+
+    @inlinable
```

Comment:
Do we really need `@inlinable` here or is `@usableFromInline` enough?  AFAICT nothing is generic and therefore doesn't really benefit from `@inlinable`

Reply:
Having everything be `@inlinable` is still useful, even without generics: it allows the compiler to observe that these functions don't affect global state, causing ARC traffic to be reduced.